nativelib/src/main/resources/gc/immix/datastructures/Stack.h

```diff
@@ -4,9 +4,7 @@
 #include "../GCTypes.h"
 #include "../headers/ObjectHeader.h"
 
-#define PRINT_STACK_OVERFLOW
-
-#define INITIAL_STACK_SIZE (8)
+#define INITIAL_STACK_SIZE (128 * 1024)
```

Comment:
This seems to be really small, is this sufficient not to overflow on our benchmarks?